src/pages/settings/SettingsPage.tsx

```diff
@@ -0,0 +1,112 @@
+import { Button, Flex, Input, Label, MainHeadline, Spacing } from '@hedvig-ui'
+import React, { useEffect, useState } from 'react'
+import { toast } from 'react-hot-toast'
+import { useGetMeQuery, useUpdateUserMutation } from 'types/generated/graphql'
+
+export const SettingsPage: React.FC<{}> = ({}) => {
+  const { data } = useGetMeQuery()
+  const [fullName, setFullName] = useState('')
+  const [phoneNumber, setPhoneNumber] = useState<null | string>('')
+  const [updateUser] = useUpdateUserMutation()
+
+  const handleSaveChanges = () => {
+    if (!data) {
+      return
+    }
+
+    toast.promise(
+      updateUser({
+        variables: {
+          input: {
+            fullName,
+            phoneNumber,
+          },
+        },
+        optimisticResponse: {
+          updateUser: {
+            __typename: 'User',
+            ...data.me.user,
+            fullName,
+            phoneNumber,
+          },
+        },
+      }),
+      {
+        loading: 'Updating user',
+        success: 'User updated',
+        error: 'Could not update user',
+      },
+    )
+  }
+
+  const reset = () => {
+    setFullName(data?.me.user.fullName ?? '')
+    setPhoneNumber(data?.me.user.phoneNumber ?? null)
+  }
+
+  useEffect(reset, [])
+
+  const changes =
+    fullName !== data?.me.user.fullName ||
+    phoneNumber !== data.me.user.phoneNumber
+
+  if (!data) {
+    return null
+  }
+
+  return (
+    <>
+      <MainHeadline>Settings</MainHeadline>
+      <Spacing top />
+      <Flex direction="column">
+        <form
+          style={{ width: '100%' }}
```

Comment:
So we don't have to repeat this over and over?
Or add a div underneath with the max-width
```suggestion
          style={{ width: '100%', maxWidth: '350px' }}
```